file-sharing/src/tabs/iSCSI/ui/screens/logicalUnitNumbers/LogicalUnitNumberEditor.vue

```diff
@@ -100,6 +118,18 @@ const createLun = () => {
     );
 };
 
+function populateNextNumber() {
+  let nextNumber = 0;
+
+  const existingNumbers = props.initiatorGroup.logicalUnitNumbers.map((lun) => lun.unitNumber);
+  
+  while (existingNumbers.includes(nextNumber)) {
+    nextNumber += 1;
+  }
+
+  tempLun.value.unitNumber = nextNumber;
+}
```

Comment:
can speed this up a lot by doing:
```ts
function populateNextNumber() {
  tempLun.value.unitNumber = Math.max(
    ...props.initiatorGroup.logicalUnitNumbers.map((lun) => lun.unitNumber)
  ) + 1;
}
```
instead of searching list N times